src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml

```diff
@@ -3,104 +3,36 @@
-    ViewBag.Title = $"Gateway - {heading}";
+    ViewBag.Title = $"Gateway - {Model.Heading}";
     Layout = "_Layout";
-
-    var formGroupError = Model.ErrorMessages != null && Model.ErrorMessages.Any() ? "govuk-form-group--error" : "";
-    var isErrorFailText = "";
-    var noSelectionMessage = "";
-    var failTextMessage = "";
-    if (Model.ErrorMessages != null && Model.ErrorMessages.Any())
-    {
-        foreach (var error in Model.ErrorMessages)
-        {
-            if (error.Field == "OptionPass")
-            {
-                noSelectionMessage = error.ErrorMessage;
-            }
-
-            if (error.Field == "OptionFailText")
-            {
-                isErrorFailText = "govuk-textarea--error-override";
-                failTextMessage = error.ErrorMessage;
-            }
-        }
-    }
 }
 
-<div class="govuk-breadcrumbs">
-    <ol class="govuk-breadcrumbs__list">
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="/Dashboard">RoATP dashboard</a>
-        </li>
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="/Roatp/Gateway/New">RoATP applications</a>
-        </li>
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="/Roatp/Gateway/@Model.ApplicationId">Application assessment overview</a>
-        </li>
-        
-        <li class="govuk-breadcrumbs__list-item" aria-current="page">@Model.PageTitle</li>
-    </ol>
-</div>
+@{ await Html.RenderPartialAsync("~/Views/Roatp/Apply/Gateway/Pages/_GatewayAssessmentBreadcrumb.cshtml"); }
 <main class="govuk-main-wrapper " id="main-content" role="main">
-
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            @if (Model.ErrorMessages != null && Model.ErrorMessages.Any())
-            {
-                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
-                    <h2 class="govuk-error-summary__title" id="error-summary-title">
-                        There is a problem
-                    </h2>
-                    <div class="govuk-error-summary__body">
-                        <ul class="govuk-list govuk-error-summary__list">
-                            @foreach (var error in Model.ErrorMessages)
-                            {
-                                <li><a href="#@error.Field">@error.ErrorMessage</a></li>
-                            }
-                        </ul>
-                    </div>
-                </div>
-            }
-
-            <span class="govuk-caption-xl">
-                @heading
-            </span>
-
-            <h1 class="govuk-heading-xl">@Model.PageTitle</h1>
-
+            @{ await Html.RenderPartialAsync("~/Views/Roatp/Apply/Gateway/Pages/_GatewayApprovalValidationSummary.cshtml"); }
```

Comment:
Use the partial tag helper as Html.RenderPartialAsync is deprecated in ASP.NET Core